x-pack/plugins/security_solution/public/explore/users/components/kpi_users/authentications/index.tsx

```diff
@@ -82,7 +82,7 @@ const UsersKpiAuthenticationsComponent: React.FC<UsersKpiProps> = ({
     skip: querySkip,
```

Comment:
can you please add the check here:
```suggestion
    skip: querySkip || isChartEmbeddablesEnabled,
```